pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/SystemUiExtension.kt

```diff
@@ -30,3 +38,41 @@ fun Activity.showSystemUi() {
         window.decorView
     ).show(WindowInsetsCompat.Type.systemBars())
 }
+
+/**
+ * Show system ui
+ *
+ * @param isShowed true to display system ui (status bar and navigation bar)
+ */
+@Composable
+fun showSystemUi(isShowed: Boolean) {
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>ComposableNaming: Composable functions that return Unit should start with an uppercase letter